test/controllers/UploadControllerSpec.scala

```diff
@@ -707,9 +707,12 @@ class UploadControllerSpec extends FrontEndTestHelper {
          |            </a>""".stripMargin
     )
     pageAsString must include(
-      """|                    <p class="upload-progress-error-timeout__message" hidden>Your upload has timed out. Click 'Return to start' to begin a new transfer.</p>
-         |                    <p class="upload-progress-error-authentication__message" hidden>You have been signed out. Click 'Return to start' to begin a new transfer.</p>
-         |                    <p class="upload-progress-error-general__message" hidden>Click 'Return to start' to begin a new transfer.</p>""".stripMargin
+      """                    <p class="upload-progress-error-timeout__message" hidden>""" +
```

Comment:
These lines will be short enough if you remove the whitespace between | and < so can you do that? I'm not sure if that will still work but it does look easier to read when they're not split out over separate lines. Don't worry if the whitespace is needed.

Reply:
Yes, I tried this but it doesn't work; the issue is the new line character directly after the `hidden>`. Maybe I could add a `.replace("hidden>\n",  "hidden>")` at the end of the string?

Reply:
No, don't worry about it if it makes it more complicated. I was just wondering.